assets/dev/js/editor/components/dynamic-tags/control-behavior.js

```diff
@@ -51,6 +51,19 @@ module.exports = Marionette.Behavior.extend( {
 		this.ui.dynamicSwitcher.removeClass( 'elementor-control-unit-1' ).addClass( 'e-control-tool' );
 
 		$colorPickerToolsContainer.append( this.ui.dynamicSwitcher );
+
+		const $colorPicker = jQuery( Marionette.Renderer.render( '#tmpl-elementor-control-element-color-picker' ) ),
+			currentElement = elementor.getPanelView().getCurrentPageView().getOption( 'editedElementView' );
+
+		$colorPicker.on( 'click', () => {
+			$e.run( 'elements-color-picker/start', {
```

Comment:
```suggestion
			$e.run( 'panel/editor/color-picker/start' ),
```

**color-picker** can be sob component of **panel/editor**

@matipojo btw: `panel/editor` should be renamed to `editor/panel` like `editor/documents`.